src/components/NewProject/index.jsx

```diff
@@ -1,4 +1,33 @@
+import { useNavigate, useSearchParams } from "react-router-dom";
+
+import { useEffect } from "react";
+import { getToken } from "../../services/auth";
+
 function NewProejct() {
```

Comment:
P1: 컴포넌트명에 오타가 있었네요. 해당 오타 수정 부탁 드립니다.

```suggestion
function NewProject() {
```

Reply:
PR시 오타 확인하겠습니다.